lib/promenade/waterdrop/error_subscriber.rb

```diff
@@ -0,0 +1,27 @@
+require "promenade/waterdrop/subscriber"
+
+module Promenade
+  module Waterdrop
+    class ErrorSubscriber < Subscriber
+      attach_to "error.waterdrop"
+
+      Promenade.counter :waterdrop_errors do
```

Comment:
Same naming suggestion as https://github.com/errm/promenade/pull/43/files#r1129265363
```suggestion
      Promenade.counter :waterdrop_errors_total do
```